extensions/cornerstone/package.json

```diff
@@ -29,7 +29,7 @@
   "peerDependencies": {
     "@ohif/core": "^3.0.0",
     "@ohif/ui": "^2.0.0",
-    "cornerstone-wado-image-loader": "^4.2.1",
+    "cornerstone-wado-image-loader": ">=4.8.0",
```

Comment:
This isn't the right import any more - should be @cornerstonejs/...wado image loader now.

Reply:
I just changed the version? Are you confusing with  "@cornerstonejs/streaming-image-volume-loader"?